gradle/libs.versions.toml

```diff
@@ -94,7 +96,7 @@ jacoco = { group = "org.jacoco", name = "org.jacoco.core", version.ref = "jacoco
 dhis2-android-sdk = { group = "org.hisp.dhis", name = "android-core", version.ref = "dhis2sdk" }
 dhis2-ruleengine = { group = "org.hisp.dhis.rules", name = "rule-engine-jvm", version.ref = "ruleEngine" }
 dhis2-expressionparser = { group = "org.hisp.dhis.lib.expression", name = "expression-parser-jvm", version.ref = "expressionParser" }
-dhis2-mobile-designsystem = { group = "org.hisp.dhis.mobile", name = "designsystem-android", version.ref = "designSystem" }
+dhis2-mobile-designsystem = { group = "org.hisp.dhis.mobile", name = "designsystem", version.ref = "designSystem" }
```

Comment:
do you remember that we changed this time ago because it wasn't getting right the snapshot version?

Reply:
yes. I found what the problem was and fix it. It was related on how the repositories are defined